utils/gen-unicode-data/Sources/GenUtils/Emit.swift

```diff
@@ -10,6 +10,22 @@
 //
 //===----------------------------------------------------------------------===//
 
+func _eytzingerize<C: Collection>(_ collection: C, result: inout [C.Element], sourceIndex: Int, resultIndex: Int) -> Int where C.Element: Comparable, C.Index == Int {
```

Comment:
Could you add a comment as to what these functions do?

Reply:
Will do :)